hawc/apps/summary/forms.py

```diff
@@ -881,8 +953,7 @@ def clean(self):
                 worksheet_names = open_workbook(file_contents=excel_file.read()).sheet_names()
             except XLRDError:
                 self.add_error(
```

Comment:
can you run `format-py` on this? I can't really review until it's been cleaned up a bit